corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -703,18 +703,20 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
-        showHideMap: function () {
+        showHideMap: function (e) {
             const mapDiv = $('#module-case-list-map');
             const moduleCaseList = $('#module-case-list');
             const hideButton = $('#hide-map-button');
             if (!mapDiv.hasClass('hide')) {
                 mapDiv.addClass('hide');
                 moduleCaseList.removeClass('col-md-7 col-md-pull-5').addClass('col-md');
```

Comment:
Looking at this again I don't know if `col-md` is technically a real Bootstrap class (it would be identical to `col` or `col-md-12`), but I think styling gets applied the same way so it doesn't strictly matter.